packages/client-core/src/transports/SocketWebRTCClientFunctions.ts

```diff
@@ -459,6 +460,8 @@ export async function onConnectToInstance(network: SocketWebRTCClientNetwork) {
   network.ready = true
 
   logger.info('Successfully connected to instance type: %o', { topic: network.topic, hostId: network.hostId })
+  console.log('network', network)
+  updateNetwork(network)
```

Comment:
can we add a todo here to avoid using this once all the properties are being reactively set